backend/src/diagnostic.rs

```diff
@@ -56,7 +57,12 @@ pub fn restart(#[context] ctx: DiagnosticContext) -> Result<(), Error> {
     Ok(())
 }
 
-#[command(rename = "forget-disk", display(display_none))]
+#[command(subcommands(forget_disk, repair))]
+pub fn disk() -> Result<(), Error> {
+    Ok(())
```

Comment:
why do we not need to `tokio::fs::write(REPAIR_DISK_PATH, b"").await?;` here?

Reply:
that does happen in the repair function which is a child of this